### ISSUE TITLE
where too put openvpn config file

### ISSUE THREAD

**Issue by icultibate420** [fswdevl/Destiny-2-SDR-Matchmaking-Firewall#21]
ive installed everything im on ubuntu desktop 21.04 and i have a xr500 router i have downloaded the file and have it on my desktop but as for where too put the file or how too set up the vpn on my router i have no clue any help would be amazing thank you!!

**Comment by xvikings** [comment]
go to hybird vpn then vpn setup then advance and add the config file there

**Comment by icultibate420** [comment]
when i do that it shows username password and configuration do i just copy and paste whats in the config file?

**Comment by xvikings** [comment]
yes copy and past it and about the user and password if you add a password to ur vpn config then add it otherwise just type anything and sorry for my bad English :P

**Comment by xvikings** [comment]
btw i would like to try something with you if your free

**Comment by icultibate420** [comment]
im up for it add my psn ill invite you too a chat ! iCultivate420

**Comment by xvikings** [comment]
do you have discord ?

**Comment by icultibate420** [comment]
yes! iCultivate420#4466

**Comment by xvikings** [comment]
ok one sec

**Comment by mCutaa** [comment]
@icultibate420 I've added you on discord. I'm running the same setup (PS & XR500). Would be keen to test it out together if you want :)
